app/jobs/calendar_importer/parsers/base.rb

```diff
@@ -48,5 +48,11 @@ def calendar_to_events
     def digest(data)
       Digest::MD5.hexdigest(data.to_s)
     end
+
+    def safely_parse_json(string, default = nil)
+      JSON.parse string
```

Comment:
I need a rails person to confirm, but should there be a `begin` before this line?

Reply:
This is valid ruby. Methods can have `rescue` blocks at the end of them. There are tests covering this specific method in this PR